CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ target_include_directories(gumbo PUBLIC
 if(WIN32)
```

Comment:
It should be so:
```
if(MSVC)
```
For example, MinGW is GNU compiler with WIN32=true, and it does not need file 'visualc/include/strings.h'.

Reply:
Fixed. Thank you.